shuffle/move/examples/main/devapi.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Generated on new project creation. Invoked by shuffle CLI.
+
+// Creates typescript wrappers around the Developer API for easier consumption,
+// including endpoints: ledgerInfo, resources, modules, and some of transactions.
+// Developer API: https://docs.google.com/document/d/1KEPnGGU3zg_RmN8V4r2ms_MFPwsTMNyK7jCUFygviDg/edit#heading=h.hesw425dw9gz
+
+// deno-lint-ignore-file no-explicit-any
+import { createRemote } from "https://deno.land/x/gentle_rpc@v3.1/mod.ts";
+import * as context from "./context.ts";
+
+console.log(await ledgerInfo());
+
+export async function ledgerInfo() {
```

Comment:
as a TODO i think we should add documentation to each function here, or link to corresponding openapi doc, but next PR